process/notifier/errors.go

```diff
@@ -3,3 +3,15 @@ package notifier
 import "errors"
 
 var errNoSubscribedAddresses = errors.New("no subscribed addresses provided")
+
+var errDuplicateSubscribedAddresses = errors.New("duplicate subscribed addresses provided")
+
+var errNilExtendedHeaderHandler = errors.New("nil extended handler provided")
+
+var errReceivedHeaderType = errors.New("received invalid header type")
```

Comment:
```suggestion
var errInvalidHeaderTypeReceived = errors.New("invalid header type received")
```
also no need for `provided` on errors

Reply:
Renamed as suggested.
Would leave provided in description as it is